// Copyright 2017 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package media;

message CapabilitiesInfo {
  required uint32 version = 1;
  required uint64 frames_decoded = 2;
  required uint64 frames_dropped = 3;
}